app/javascript/components/student/tracks-list/TagOptionList.jsx

```diff
@@ -17,25 +18,32 @@ export function TagOptionList({
 
   return (
     <form onSubmit={onSubmit}>
-      {options.map((option) => {
-        return (
-          <div key={option.category}>
-            <p>{option.category}</p>
-            {option.options.map((option) => {
-              return (
-                <TagOption
-                  key={option.value}
-                  onChange={handleChange}
-                  checked={selectedTags.includes(option.value)}
-                  label={option.label}
-                  value={option.value}
-                />
-              )
-            })}
-          </div>
-        )
-      })}
-      <button>Apply</button>
+      <div className="categories">
+        {options.map((option) => {
+          return (
+            <div key={option.category} className="category">
+              <h4>{option.category}</h4>
+              {option.options.map((option) => {
+                return (
+                  <TagOption
+                    key={option.value}
+                    onChange={handleChange}
+                    checked={selectedTags.includes(option.value)}
+                    label={option.label}
+                    value={option.value}
+                  />
+                )
+              })}
```

Comment:
I think this can be:

```suggestion
              {option.options.map((option) => (
                  <TagOption
                    key={option.value}
                    onChange={handleChange}
                    checked={selectedTags.includes(option.value)}
                    label={option.label}
                    value={option.value}
                  />
                )
              )}
```

I can't verify as there is a compile error in the JS.

Reply:
Thanks!

Reply:
You are correct